bokehjs/src/lib/core/util/date.ts

```diff
@@ -0,0 +1,14 @@
+export function format_date_rfc7231(date: Date): string {
```

Comment:
I would add a link to the relevant RFC section for the sake of completeness:
```suggestion
export function format_date_rfc7231(date: Date): string {
  /**
   * See https://datatracker.ietf.org/doc/html/rfc7231#section-7.1.1.1.
   */
```

Reply:
There is no need to implement this, because the `Date` object has a function [`toUTCString()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toUTCString) which returns the RFC 7231 format.

Reply:
Thank you for the confirmation on this, I had been unsure of the standard toUTCString() follows. ECMA-262 lists toUTCString() to return RFC 7231. I will make this change.